docs/integrations/lightgbm.rst

```diff
@@ -3,33 +3,85 @@
 Neptune-LightGBM Integration
 ============================
 
-|lightGBM| is a popular gradient boosting library. The integration with Neptune lets you log training and evaluation metrics and have them visualized in Neptune.
+|Run on Colab|
```

Comment:
@SiddhantSadangi could you please update this video (as we've updated the colab). 
Once we have that we are ready to merge.